WalletWasabi.Gui/CommandLine/MixerCommand.cs

```diff
@@ -19,7 +19,7 @@ public MixerCommand(Daemon daemon)
 				"Start mixing without the GUI with the specified wallet.",
 				"eg: ./wassabee mix --wallet:MyWalletName --keepalive",
 				{ "h|help", "Displays help page and exit.", x => ShowHelp = x != null },
-				{ "w|wallet:", "The name of the wallet file.", x => WalletName = x },
+				{ "w|wallet=", "The name of the wallet file.", x => WalletName = x },
 				{ "destination:", "The name of the destination wallet file.", x => DestinationWalletName = x },
```

Comment:
`destination` is also required. You cannot pass the `--destination` without a wallet.

```suggestion
				{ "destination=", "The name of the destination wallet file.", x => DestinationWalletName = x },
```